pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -193,15 +188,12 @@ in
     meta.platforms = [ "x86_64-linux" "aarch64-linux" ];
   };
 
-  beetle-saturn-hw = mkLibRetroCore {
```

Comment:
This core never worked as it should: https://github.com/libretro/libretro-core-info/issues/8

However I am not sure how to deprecate this core correctly.